scripts/ovpsim_log_to_trace_csv.py

```diff
@@ -27,34 +27,35 @@
 from riscv_trace_csv import *
 
 try:
-    from ovpsim_log_to_trace_csv_vectors import *
+  from ovpsim_log_to_trace_csv_vectors import *
 except:
-    def assign_operand_vector(a,b,c,d):
-        """ stub version when no vector processing included """
-        logging.info("No OVPsim vector instruction processing included")
-        if stop_on_first_error:
-            sys.exit(-1)
-    def is_an_extension_instruction(instr):
-        if 'v' in instr[0]:
-            return True
-        return False
+  def assign_operand_vector(a,b,c,d):
+    """ stub version when no vector processing included """
+    logging.info("No OVPsim vector instruction processing included")
+    if stop_on_first_error:
+      sys.exit(-1)
```

Comment:
What about if we can use enum in script/lib instead of the number ?. Ex:
RET_SUCCESS = 1
RET_FAIL = 0
RET_FATAL = -1

Reply:
Good idea, I think that might simplify things. I've added an initial implementation of this enums, PTAL